session4/src/com/itschool/homework/Challenge4.java

```diff
@@ -0,0 +1,33 @@
+package com.itschool.homework;
+
+public class Challenge4 {
+
+    public static void main(String[] args) {
+
+        int a = 1;
+
+        System.out.println(a++);
+        System.out.println(a++);
+        System.out.println(a++);
+        System.out.println(a++);
+        System.out.println(a++);
+        System.out.println(a++);
+        System.out.println(a++);
+        System.out.println(a++);
+        System.out.println(a++);
+        System.out.println(a);
+        System.out.println(--a);
+        System.out.println(--a);
+        System.out.println(--a);
+        System.out.println(--a);
+        System.out.println(--a);
+        System.out.println(--a);
+        System.out.println(--a);
+        System.out.println(--a);
+        System.out.println(--a);
```

Comment:
this is indeed a way to do it with what we learned so far, but what if we want to count up to 1000? :)
In the next lessons we will learn how to achieve this with repetitive loops.